### DISCUSSION TITLE
2.0.1 RPM for EL8

### DISCUSSION THREAD

**Discussion by tendonut** [openzfs/zfs#11506] [Q&A]
I got bit by doing a fresh install of CentOS Stream 8, which currently runs the 4.18.0-257.el8 kernel, but discovered that has a compatibility issue with the 2.0.0 and 0.8.6 ZFS DKMS module releases. This was fixed in the 2.0.1 release [here](https://github.com/openzfs/zfs/pull/11374) but that release hasn't been pushed to the EL8 repo yet. 

When is that expected to happen?  I'm currently running an older CentOS 8 kernel I installed manually so I can get my ZFS array up and running.

**Comment by behlendorf** [comment]
Thanks for letting us know.  It looks like that dkms repository was accidentally overlooked, we'll see about getting it updated.  In the meanwhile you can [build the packages](https://openzfs.github.io/openzfs-docs/Developer%20Resources/Custom%20Packages.html#dkms) yourself from the official 2.0.1 tarball if you prefer.
